test/e2e/common/configmap.go

```diff
@@ -133,6 +133,26 @@ var _ = Describe("[sig-node] ConfigMap", func() {
 		configMap, err := newConfigMapWithEmptyKey(f)
 		framework.ExpectError(err, "created configMap %q with empty key in namespace %q", configMap.Name, f.Namespace.Name)
 	})
+
+	/*
+	   Release : v1.16
+	   Testname: Update ConfigMap
+	   Description: Attempt to update a ConfigMap after create one.
+	*/
+	framework.ConformanceIt("should patch ConfigMap successfully", func() {
+		name := "configmap-test-" + string(uuid.NewUUID())
+		configMap := newConfigMap(f, name)
+		ginkgo.By(fmt.Sprintf("Creating configMap %v/%v", f.Namespace.Name, configMap.Name))
+		_, err := f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Create(configMap)
+		framework.ExpectNoError(err)
+
+		configMap.Data = map[string]string{
+			"data": "value",
+		}
+		ginkgo.By(fmt.Sprintf("Updating configMap %v/%v", f.Namespace.Name, configMap.Name))
+		_, err = f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Update(configMap)
+		framework.ExpectNoError(err)
```

Comment:
I would expect to see additional verification that the update took effect (get after update).  As written this test would pass against an API that 200 OK'ed everything but didn't actually update any data.

Reply:
@BobyMCbobs and I are working on implementing a verification step. Will open a new PR and link to this when ready.

Reply:
Yeah, it is nice to check the Update of `"data": "value"` is affected by Getting configmap after line 154.

Reply:
Please see https://github.com/kubernetes/kubernetes/pull/80787 for our update addressing this.